static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -69,9 +86,11 @@ function isRRWebEventAttachment(attachment: IssueAttachment) {
 }
 
 const INITIAL_STATE: State = Object.freeze({
+  errors: undefined,
   event: undefined,
   fetchError: undefined,
   fetching: true,
+  isErrorsFetching: true,
```

Comment:
@NisanthanNanthakumar the hook handles data fetching, so its initial state should be loading because it hasn't made a request for data yet. Otherwise it'd be in a weird, not loading, no error, no data state